packages/grafana-data/src/field/fieldOverrides.ts

```diff
@@ -254,20 +247,21 @@ export function setFieldConfigDefaults(config: FieldConfig, defaults: FieldConfi
 const processFieldConfigValue = (
   destination: Record<string, any>, // it's mutable
   source: Record<string, any>,
-  key: string,
+  fieldConfigProperty: FieldPropertyEditorItem,
```

Comment:
think FieldPropertyEditorItem should be renamed to FieldConfigPropertyItem, as it's more than just an editor, is really a descriptor for a Field Config Property. but very important, but something I was thinking when working on the previous PR